### PR TITLE
wip: tests: update lantern test scripts to use new golden format

### PR DESCRIPTION
**Pull request by connorjclark**
Tweaked the structure of `golden.json` a bit in #9662, so I had to update the scripts for `yarn test-lantern`.

### CODE REVIEW

File: lighthouse-core/scripts/lantern/print-correlations.js [hunk 1]
```diff
@@ -339,13 +319,13 @@ evaluateAndPrintAccuracy('firstMeaningfulPaint', 'optimisticFMP');
...
-evaluateAndPrintAccuracy('timeToFirstInteractive', 'pessimisticTTFCPUI');
-evaluateAndPrintAccuracy('timeToFirstInteractive', 'roughEstimateOfTTFCPUI');
+evaluateAndPrintAccuracy('interactive', 'optimisticTTFCPUI');
+evaluateAndPrintAccuracy('interactive', 'pessimisticTTFCPUI');
+evaluateAndPrintAccuracy('interactive', 'roughEstimateOfTTFCPUI');
 
-evaluateAndPrintAccuracy('timeToConsistentlyInteractive', 'optimisticTTI');
-evaluateAndPrintAccuracy('timeToConsistentlyInteractive', 'pessimisticTTI');
-evaluateAndPrintAccuracy('timeToConsistentlyInteractive', 'roughEstimateOfTTI');
+evaluateAndPrintAccuracy('firstCPUIdle', 'optimisticTTI');
```
Comment (patrickhulce) [inline lighthouse-core/scripts/lantern/print-correlations.js]: fyi these two are backwards :)

**Comment by connorjclark** [comment]
> I thought you just wanted to add a new golden format for the incoming stuff in the zips, not change everything historical we have so far

That's right. But these test scripts use the exact data structure that the collection process emits. The changes aren't big (nixed toplevel `sites`, `wpt3g` -> `wpt`, `wpt` also indexes outputs from WPT (not just metrics), `metrics` property that is exactly what the `metrics` audit returns instead of the deprecated names that are currently used).

I didn't think it was important to keep the same exact structure, and I already was able to make use of the better format to [analyze](https://github.com/GoogleChrome/lighthouse/pull/9706#discussion_r329243888) something (although it was inconclusive).

> can we just add LCP for early analysis to existing format and hold off hardening until we've concluded the initial work?

How about updating the `golden.js` script in #9662 to emit the data structure these scripts expect, but keep the newer format for the collection of all traces (what `collect.js` emits)?

File: lighthouse-core/scripts/lantern/print-correlations.js (hunk 1 above)
Comment (connorjclark) [reply lighthouse-core/scripts/lantern/print-correlations.js]: oopsie

**Comment by patrickhulce** [comment]
> How about updating the golden.js script in #9662 to emit the data structure these scripts expect, but keep the newer format for the collection of all traces (what collect.js emits)?

That sounds great, that's kinda what I thought was going to happen anyway :) (at least some translation script between the new golden format to old golden format temporarily)

**Pull request closed without merging**